server/lib/konnector_poller.coffee

```diff
@@ -18,187 +18,154 @@ month = 30 * day
 format = "DD/MM/YYYY [at] HH:mm:ss"
 periods = {hour: hour, day: day, week: week, month: month}
 timeouts = {}
+nextUpdates = {}
+timeout = null
 
 class KonnectorPoller
 
-    start: ->
+    start: (reset=false, cb=null) ->
```

Comment:
I would use `callback` here (see my other comment on the current "callback" variable)